apps/api/e2e/api/environment/regenerate-api-keys.e2e.ts

```diff
@@ -0,0 +1,21 @@
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
+
+describe('Environment - Regenerate Api Key', async () => {
+  let session: UserSession;
+
+  before(async () => {
+    session = new UserSession();
+    await session.initialize();
+  });
+
+  it('should regenerate an Api Key', async () => {
+    const { body: oldApiKeys } = await session.testAgent.get<{ key: string }[]>('/v1/environment/api-keys');
+    const oldApiKey = oldApiKeys[0].key;
+
+    const { body: newApiKeys } = await session.testAgent.get<{ key: string }[]>('/v1/environment/api-keys/regenerate');
+    const newApiKey = newApiKeys[0].key;
+
+    expect(oldApiKey).to.not.equal(newApiKey);
```

Comment:
Would be great if we could test the new key using it in an api request just to make sure so regenerate not breaks the usage of the api 😄

Reply:
Will do 👍

Reply:
Honestly I don't know how to do it, feel free to edit the PR if you don't mind